internal/metrics.go

```diff
@@ -53,13 +53,15 @@ func PushMetrics() {
 		return
 	}
 
-	err := pushMetrics(address)
+	extraTags := viper.GetStringMapString(StatsdExtraTagsSetting)
```

Comment:
Just digging a bit into what viper does with this - I don't *think* we can easily populate a `StringMapString` from env or cli, so perhaps this would be better as `GetString`, the breaking down something like `foo=bar,baz=qux`? I don't see any other users of `GetStringMapString` in wal-g, anyway. Happy with whatever works here.